code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1505,6 +1505,19 @@
 	L.adjustToxLoss(-5 * REM * delta_time, FALSE)
 	L.adjustBruteLoss(-2 * REM * delta_time, FALSE)
 
+/datum/reagent/pluoxium
+	name = "Pluoxium"
+	description = "An Oxygen compound that delivers eight times more Oxygen"
+	reagent_state = GAS
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+	color = "90560B"
+	taste_description = "like air but better"
+	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED|REAGENT_NO_RANDOM_RECIPE
+
+/datum/reagent/pluoxium/on_mob_life(mob/living/Lmob, delta_time, times_fired)
+	. = ..()
+	Lmob.adjustOxyLoss(-2 * REM * delta_time, FALSE)
```

Comment:
please don't call it Lmob. 1 because capitals in var names do nothing, and 2 because it says literally nothing that the typepath doesn't say.
You want var names to convey more information then just the type, saying something about what the mob "is" in the context of this proc. Are we inside the mob, is the mob being splashed, are we burning the mob alive, etc